lessons/04-client-jquery-ajax/HOMEWORK.md

```diff
@@ -0,0 +1,11 @@
+### Class 4 Homework
+For the homework, you will be integrating mongoDB into your Cat App from last class using mongoose. Now you can store the cats you create forever, and create new features that allow you to sort those cats in various ways.
+
+For this homework, you will need to add at least one feature that integrates with your database and uses an advanced query. For example, you could only show cats in a certain age range, or display cats by date created.
```

Comment:
Definitely in mongoose as they are integrating it into the application. I think that is a good way to define advanced query, unless anyone had something else in mind.